src/LookupTable/components/list/ListTable.tsx

```diff
@@ -2,30 +2,20 @@ import React from 'react'
 import { NavLink } from 'react-router-dom'
 import { Button, Header, Icon, Message, Popup, Table } from 'semantic-ui-react'
 import { Loading } from '../../../components'
-import { FieldMapType, LookUpTableType } from '../../types'
+import { LookUpTableType } from '../../types'
 import { DownloadButton } from '..'
 import { useRouter } from '../../../utils/hooks/useRouter'
 import { useLanguageProvider } from '../../../contexts/Localisation'
 
 const TABLE_PREFIX = 'data_table_'
 
-const ListTable: React.FC<any> = ({
-  allTableStructures,
-  allTableStructuresLoadState,
-  setAllTableStructures,
-}: any) => {
+const ListTable: React.FC<any> = ({ allTableStructures, allTableStructuresLoadState }: any) => {
   const { t } = useLanguageProvider()
   const { loading, error } = allTableStructuresLoadState
   const {
     match: { path },
   } = useRouter()
 
-  const handleExpansion = (lookupTable: LookUpTableType) => {
-    if (!lookupTable) return
-    lookupTable.isExpanded = !lookupTable.isExpanded
-    setAllTableStructures(allTableStructures)
-  }
-
```

Comment:
I had no idea this "handleExpansion" even existed. The logic was broken so we'd never actually seen it in the UI when clicking the rows. I turned it on to check it, doesn't seem useful so have just removed it rather than trying to make it work properly.